sdk/identity/azure-identity/src/main/java/module-info.java

```diff
@@ -10,6 +10,7 @@
     requires com.sun.jna.platform;
     requires org.reactivestreams;
     requires java.xml;
+    requires java.sql;
```

Comment:
Was this added for testing purposes? If so, is there another way we could ensure this is available without making it part of our shipped library?

Reply:
Yeah, there was a usage in one of the tests. I hit issues trying to build without putting this here. I've changed the test to use `java.util.Date` directly.